pycbc/transforms.py

```diff
@@ -2501,3 +2501,61 @@ def read_transforms_from_config(cp, section="transforms"):
         t = transforms[name].from_config(cp, section, subsection)
         trans.append(t)
     return order_transforms(trans)
+
+
+def new_z_to_euler(new_z):
+    """
+    From the new Z axis expressed in (polar, azimuthal) angles of the
+    initial coordinate system, return the (alpha, beta) Euler angles
+    that rotate the old Z axis (0, 0) to the new Z axis.
+    """
+    return (0.5*numpy.pi + new_z[1]) % (2 * numpy.pi), new_z[0]
+
+
+def rotate_euler(sph_coords, alpha, beta, gamma):
+    """
+    Take an Nx2 array of N (theta, phi) vectors on the unit sphere
+    (that is, (polar, azimuthal) angles in radians) and apply
+    rotations through the Euler angles alpha, beta, and gamma
+    in radians, using the ZXZ convention.
+    """
+    c = numpy.cos
+    s = numpy.sin
+
+    # Define rotation matrix
+    R = numpy.array(
```

Comment:
I think scipy can quickly produce rotation matrices. Is this providing functionality not already defined there? This may be fine as is, but just wanted to ask in case just using scipy to get the rotation matrices would be cleaner.